rabbitmq/json-overview.yml

```diff
@@ -0,0 +1,180 @@
+type: json
+data:
+  url: /api/overview
+  server: ${SPM_MONITOR_RABBITMQ_API_URL}
+  basicHttpAuthUsername: ${SPM_MONITOR_RABBITMQ_BASICAUTH_USERNAME}
+  basicHttpAuthPassword: ${SPM_MONITOR_RABBITMQ_BASICAUTH_PASSWORD}
+
+observation:
+  - name: overview
+    metricNamespace: rabbitmq
+    path: $.
+
+    metric:
+      - name: object_totals.connections
+        source: object_totals/connections
+        type: long_gauge
+        label: connections
+        description: Total number of connections
+
+      - name: object_totals.channels
+        source: object_totals/channels
+        type: long_gauge
+        label: free space
+        unit: channels
+        description: Total number of channels
+
+      - name: object_totals.queues
+        source: object_totals/queues
+        type: long_gauge
+        label: queues
+        description: Total number of queues
+
+      - name: object_totals.consumers
+        source: object_totals/consumers
+        type: long_gauge
+        label: consumers
+        description: Total number of consumers
+
+      - name: queue_totals.messages.count
+        source: queue_totals/messages
+        type: long_gauge
+        label: messages
+        description: Total number of messages (ready plus unacknowledged)
+
+      - name: queue_totals.messages.rate
+        source: queue_totals/messages_details/rate
+        type: double_gauge
+        label: messages rate
+        unit: message/second
+        description: Rate of number of messages (ready plus unacknowledged)
+
+      - name: queue_totals.messages_ready.count
+        source: queue_totals/messages_ready
+        type: long_gauge
+        label: ready messages
+        description: Number of messages ready for delivery
+
+      - name: queue_totals.messages_ready.rate
+        source: queue_totals/messages_ready_details/rate
+        type: double_gauge
+        label: ready messages rate
+        unit: message/second
+        description: Rate of number of messages ready for delivery
+
+      - name: queue_totals.messages_unacknowledged.count
+        source: queue_totals/messages_unacknowledged
+        type: long_gauge
+        label: unacknowledged messages
+        description: Number of unacknowledged messages
+
+      - name: queue_totals.messages_unacknowledged.rate
+        source: queue_totals/messages_unacknowledged_details/rate
+        type: double_gauge
+        label: unacknowledged messages rate
+        unit: message/second
+        description: Rate of number of unacknowledged messages
+
+      - name: message_stats.ack
+        source: message_stats/ack
+        type: long_gauge
```

Comment:
Gauge? There are more like this one in this file.

Reply:
counter